From eeb1f45216eced42b386d6ea65eadc49798c7174 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Thu, 28 May 2020 14:00:34 +0200
Subject: [PATCH] test_obj: Ignore __class_getitem__

Ignore __class_getitem__ special that was added to some stdlib objects
in Python 3.9.  It is used as part of type declarations.
---
 tests/module/test_obj.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tests/module/test_obj.py b/tests/module/test_obj.py
index 03b100c..43c5c47 100644
--- a/tests/module/test_obj.py
+++ b/tests/module/test_obj.py
@@ -23,7 +23,8 @@ class TestDelayedInstantiation:
         def assertKls(cls, ignores=(),
                       default_ignores=("__new__", "__init__", "__init_subclass__",
                                        "__getattribute__", "__class__",
-                                       "__getnewargs__", "__doc__")):
+                                       "__getnewargs__", "__doc__",
+                                       "__class_getitem__")):
             required = set(x for x in dir(cls)
                            if x.startswith("__") and x.endswith("__"))
             missing = required.difference(obj.kls_descriptors)
-- 
2.26.2

